vars/rollingUpgradePipeline.groovy

```diff
@@ -42,52 +42,127 @@ def call(Map pipelineParams) {
+                                                        export SCT_POST_BEHAVIOR_LOADER_NODES="${params.post_behavior_loader_nodes}"
+                                                        export SCT_POST_BEHAVIOR_MONITOR_NODES="${params.post_behavior_monitor_nodes}"
+                                                        export SCT_INSTANCE_PROVISION=${pipelineParams.params.get('provision_type', '')}
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$GIT_BRANCH | sed -E 's+(origin/|origin/branch-)++')
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$SCT_AMI_ID_DB_SCYLLA_DESC | tr ._ - | cut -c1-8 )
+
+                                                        export SCT_GCE_IMAGE_DB=${pipelineParams.gce_image_db}
+                                                        export SCT_SCYLLA_LINUX_DISTRO=${pipelineParams.linux_distro}
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC="\$SCT_AMI_ID_DB_SCYLLA_DESC-\$SCT_SCYLLA_LINUX_DISTRO"
+
+                                                        export SCT_WORKAROUND_KERNEL_BUG_FOR_IOTUNE=${pipelineParams.workaround_kernel_bug_for_iotune}
+
+                                                        echo "start test ......."
+                                                        ./docker/env/hydra.sh run-test ${pipelineParams.test_name} --backend ${params.backend}  --logdir /sct
+                                                        echo "end test ....."
+                                                        """
+                                                    } catch (error) {
+                                                        sh """echo ${error}"""
+                                                    }
+                                                }
+                                            }
+                                        }
+                                        stage("Send email for Upgrade from ${base_version}") {
+                                            wrap([$class: 'BuildUser']) {
+                                                dir('scylla-cluster-tests') {
+                                                    def email_recipients = groovy.json.JsonOutput.toJson(pipelineParams.get('email_recipients', 'qa@scylladb.com'))
+                                                    try {
+                                                        sh """
+                                                        #!/bin/bash
+
+                                                        set -xe
+                                                        env
+
+                                                        echo "Start send email ..."
+                                                        ./docker/env/hydra.sh send-email --logdir /sct --email-recipients "${email_recipients}"
```

Comment:
If send_email is set to false, this cmd will be executed, but email won't be sent. right?

Reply:
This command don't process email_sent parameter. It always send email. Only one thing, will stop to send: if the email_data.json file will not be generated due to earlier testrun interruption. But this will be fixed soon.